src/options.js

```diff
@@ -98,6 +99,7 @@ function normalize(options) {
     delete normalized.useFlowParser;
   }
 
+  debugger;
```

Comment:
debugger line

Reply:
I can't even sneak this past you in someone else's project, @dule.  :)